### PR TITLE
Documentation/update readme

### PR DESCRIPTION
**Pull request by EdgarACarneiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # ExpressJS Template
 
+| INDEX |
```
Comment (cyrilico) [inline README.md]: wtf

**Comment by diogotorres97** [comment]
@EdgarACarneiro  please extract the [Handling pgadmin](https://github.com/diogotorres97/expressjs-template/tree/documentation/update-readme) to another .md file bc it's not the main focus of this repository.
Another things:
- Upgrade the index UI ( I personally don't like it)
- Remove de subsections list below the [Setup Project}(https://github.com/diogotorres97/expressjs-template/tree/documentation/update-readme)

File: README.md [hunk 2]
```diff
@@ -54,10 +65,52 @@ $ npm run lint # run linter
...
+| __Seeders__ | Definition of an initial set of values used to populate the database. |
+| __Services__ | Additional services such as _OAuth_, email integration, websockets, among others. |
+| __Test__ | Definition of a set of files with the purpose of verifying that our server is working as it is supposed to. |
+| __Utils__ | Definition of a set of utility functions that can be used across all afore-mentioned sections |
+
+Other important files:
+
+| File | Description |
+|:-:|:-|
+| __Dockerfile__| Text document that contains all the commands a user could call on the command line to assemble a docker image. |
```
Comment (cyrilico) [inline README.md]: @diogotorres97 will certainly not approve of this definition

**Comment by antonioalmeida** [comment]
> * Upgrade the index UI ( I personally don't like it)

I would remove the index table completely

File: README.md (hunk 2 above)
Comment (diogotorres97) [reply README.md]: @EdgarACarneiro This definition is quite generic (used by the [docker documentation](https://docs.docker.com/engine/reference/builder/)) and in this case should be something more specific to the project. For example, talk about the image used and the dependencies to support bcrypt.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
 # ExpressJS Template
 
+| INDEX |
+|:-:|
+| [Setup Project](#Setup-Project) |
+| [Project Architecture Overview](#Project-Architecture-Overview) |
```
Comment (rendoir) [inline README.md]: The index should be a numbered list with hrefs that point to the titles/subtitles.

File: README.md [hunk 4]
```diff
@@ -35,8 +38,16 @@ $ docker-compose up
...
 
 Access to:
-- http://localhost:3000 -> to access the server
-- http://localhost:5050 -> to access pgadming to interact with database
+- `http://localhost:3000` -> to access the server
+- `http://localhost:5050` -> to access pgadmin to interact with database
+
+### Handling pgadmin
+
+Notice that there is not yet a server created, so when you access the pgadmin interface you must create a new one. When doing so, do not forget to consult the docker-compose file (`docker-compose.yml`). To ease your creation of the server, the example below is given:
```
Comment (rendoir) [inline README.md]: You don't explain why I shouldn't "forget to consult the docker-compose file".

File: README.md [hunk 5]
```diff
@@ -35,8 +38,16 @@ $ docker-compose up
...
+
+![Example server creation](https://i.imgur.com/zeK6HfM.png)  
+
+To access the your server schemas navigate to `Servers/<server-name>/Databases/<Maintenance-database-name>/Schemas/public/Tables`.
```
Comment (rendoir) [inline README.md]: Simple typo 'To access the your server' -> 'To access your server'

File: README.md [hunk 6]
```diff
@@ -1,11 +1,14 @@
 # ExpressJS Template
 
+| INDEX |
+|:-:|
+| [Setup Project](#Setup-Project) |
+| [Project Architecture Overview](#Project-Architecture-Overview) |
+
 ## Product View
 
 Expressjs + Passportjs + JWT + Postgres + Pgadmin + Docker
```
Comment (rendoir) [inline README.md]: Missing links to their website/documentation

File: README.md (hunk 2 above)
Comment (EdgarACarneiro) [reply README.md]: @diogotorres97 Since I'm not that comfortable with the express template, I believe you should do it.

File: README.md (hunk 3 above)
Comment (EdgarACarneiro) [reply README.md]: @rendoir Would you mind making those changes for me please?

File: README.md (hunk 2 above)
Comment (diogotorres97) [reply README.md]: Dockerfile - Text document that contains all the commands used to support the express framework. It's based on `node:11-alpine` and have some dependencies installed to support bcrypt package.

File: README.md [hunk 7]
```diff
@@ -54,10 +65,52 @@ $ npm run lint # run linter
...
+| __Services__ | Additional services such as _OAuth_, email integration, websockets, among others. |
+| __Test__ | Definition of a set of files with the purpose of verifying that our server is working as it is supposed to. |
+| __Utils__ | Definition of a set of utility functions that can be used across all afore-mentioned sections |
+
+Other important files:
+
+| File | Description |
+|:-:|:-|
+| __Dockerfile__| Text document that contains all the commands a user could call on the command line to assemble a docker image. |
+| __package.json__ | _JSON_ document that describes the behavior of the related npm-package. Defines some standard information regarding it, as well as useful scripts. |
```
Comment (diogotorres97) [inline README.md]: | JSON document that describes the behavior of  the related npm-package. Defines some useful scripts used in this template but also the dependencies installed as well as their version.

**Pull request merged**